### ISSUE TITLE
Issues with .gif files (some work, others don't)

### ISSUE THREAD

**Issue by deroverda** [gildas-lormeau/SingleFileZ#156]
For example, saving [this](http://www.milosoftware.com/en/index.php?body=sawards.php) page successfully downloads 2 gifs while three of them all show size "0" and won't open when inspecting the .zip. 

Here's the two gifs that work. 
[1. 13.gif](https://www.top4download.com/templates/T4D/images/award_5.gif)
[2. 11.gif](https://www.x64bitdownload.com/templates/X64/images/award_120x60_listed.gif)

So to summarize, SingleFileZ only manages to download 3 images (2gif, 1png) and fails 3 gifs when all images (gifs) have working url's.  

What is causing this? My settings perhaps?

**EDIT:** using latest Firefox, addon downloaded through ff addon browser page. Singlefilez settings kept at default.

**Comment by gildas-lormeau** [comment]
I'm not able to reproduce your issue. On my end, all the images are present in the page and the zip file.

**Comment by deroverda** [comment]
Sorry for the late reply. Thanks for testing and confirming it works as it should. I must have done something wrong.
